examples/lib/stories/controls/scroll.dart

```diff
@@ -30,9 +28,17 @@ class ScrollGame extends BaseGame with ScrollDetector {
   void update(double dt) {
     super.update(dt);
     final target = this.target;
+    final ds = speed * dt;
     if (target != null) {
-      final dir = (target - position).normalized();
-      position += dir * (speed * dt);
+      if (position != target) {
+        final diff = target - position;
+        if (diff.length < ds) {
+          position.setFrom(target);
+        } else {
+          diff.scaleTo(ds);
+          position.setFrom(position + diff);
+        }
+      }
```

Comment:
this is the correct implementation to avoid stuttering, I will probably make an ext func later

Reply:
https://github.com/flame-engine/flame/pull/746